### ISSUE TITLE
FP in 'Unrestricted SQL Server Access' query for Terraform

### ISSUE THREAD

**Issue by t3mi** [Checkmarx/kics#3444]
### Platform
Terraform

### Provider
Azure

### Description
Following code produces false positive as '0.0.0.0' for start and end is to allow communication from all [Azure-internal IP addresses](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/sql_firewall_rule#end_ip_address).
```hcl
resource "azurerm_sql_firewall_rule" "internal_azure_resources" {
  name                = "InternalAzure"
  resource_group_name = azurerm_resource_group.test.name
  server_name         = azurerm_sql_server.test.name
  start_ip_address    = "0.0.0.0"
  end_ip_address      = "0.0.0.0"
}
```
